cni/network/network.go

```diff
@@ -736,6 +769,8 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 	}
 
 	if opt.resultV6 != nil {
+		// inject ipv6 routes to container if it's dualstack
+		epInfo.IPV6Mode = "dualstack"
```

Comment:
I see in the code, 
> IPV6Mode:           opt.nwCfg.IPV6Mode,

and this code is overwriting that value. we may need to create a new field for specifying if the endpoint is dualstack enabled. @tamilmani1989